src/main/java/cc/hyperium/mods/playtime/PlayTimeCommand.java

```diff
@@ -4,11 +4,12 @@
 import cc.hyperium.commands.BaseCommand;
 import cc.hyperium.utils.ChatColor;
 
-import java.time.Duration;
 import java.util.Collections;
 import java.util.List;
 
 public class PlayTimeCommand implements BaseCommand {
+    PlayTime playTime = new PlayTime();
```

Comment:
Both commands have this, a new instance of the mod shouldn't be made for each command

Add the mod instance to the command via a constructor which requires the mod instance

Reply:
i believe i fixed this

Reply:
I've fixed it properly, take note of the change. 

That's the standard, class instances should be reused not recreated.